pkg/controller/plan/kubevirt.go

```diff
@@ -97,6 +100,35 @@ type KubeVirt struct {
 	Builder adapter.Builder
 }
 
+type Domain struct {
```

Comment:
since it's only OVA, i would prefer it to be somewhere in its adapter but let's see what others think